sshtunnel.py

```diff
@@ -458,43 +464,46 @@ def make_ssh_forward_server(self, remote_address, local_bind_address):
     def __init__(
             self,
             ssh_address_or_host=None,
-
+            ssh_config_file="~/.ssh/config",
```

Comment:
Use `ssh_config_file` and `use_ssh_config` is a overhead. I think need to leave one. Use `ssh_config_file=None` by default. It seems that this change needs to be discussed and not a part of this code review.